specification/servicenetworking/ServiceNetworking.Management/main.tsp

```diff
@@ -87,6 +94,60 @@ model AssociationSubnet {
   id: string;
 }
 
+//----------------------- SecurityPolicy -----------------------
+@doc("SecurityPolicy Subresource of Traffic Controller.")
+@parentResource(TrafficController)
+model SecurityPolicy is TrackedResource<SecurityPolicyProperties> {
```

Comment:
nit: moving resources to separate files for better organization and easier review.

Reply:
Will definitely try and incorporate in future versions